{"uuid": "6b4a4e0f-79fd-4e59-b5e2-2e716c85e074", "children": ["06985d10-4579-4a9d-b62a-b82bbbeb3738"], "befores": [{"name": "handler", "status": "broken", "statusDetails": {"message": "OSError: [WinError 87] 参数错误。\n", "trace": "  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\po_jrj\\cases\\conftest.py\", line 9, in handler\n    driver = webdriver.Chrome('')  #调取chrom浏览器\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 73, in __init__\n    self.service.start()\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 76, in start\n    stdin=PIPE)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\subprocess.py\", line 709, in __init__\n    restore_signals, start_new_session)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\subprocess.py\", line 997, in _execute_child\n    startupinfo)\n"}, "start": 1609507722618, "stop": 1609507722621}], "start": 1609507722618, "stop": 1609507722897}